app/models/data_object.rb

```diff
@@ -80,7 +80,10 @@ class DataObject < ActiveRecord::Base
 
   scope :images, -> { where(data_type_id: DataType.image.id) }
   scope :texts,  -> { where(data_type_id: DataType.text.id) }
-
+  scope :articles_of_today, ->(current_user) {
+    DataObject.joins(:users_data_object).where(users_data_objects: {user_id: current_user.id},data_type_id: DataType.text.id,
+    created_at: DateTime.now.in_time_zone.to_date.beginning_of_day..DateTime.now.in_time_zone.to_date.end_of_day)
```

Comment:
Line is too long. [113/80]<br>Align the elements of a hash literal if they span more than one line.